spec/support/feature_spec_helpers.rb

```diff
@@ -0,0 +1,12 @@
+module FeatureSpecHelpers
+  def sign_in(user)
+    visit '/admin/signin'
+    fill_in :email, with: user.email
+    fill_in :password, with: 'password123'
+    click_button 'Continue'
```

Comment:
No, just signing in with the user passed into the method

Reply:
I think this isn't a good test then because even if it blew up after clicking continue the test may still pass. You should assert the next URL or some text on the homepage.

Reply:
This isn't testing the sign in journey itself. We use it in the user and register feature spec to just sign into the admin UI and then go from there.

Reply:
Ahhh, OK makes sense 👍